code/game/objects/items/weapons/shields.dm

```diff
@@ -48,6 +48,26 @@
 	burn_state = FLAMMABLE
 	block_chance = 30
 
+/obj/item/weapon/shield/riot/makeshift
+	name = "makeshift shield"
+	desc = "A tray with cable tightly wrapped around it to act as a handle."
+	icon_state = "makeshift"
+	item_state = "makeshift"
+	force = 5
+	block_chance = 15
+	materials = list()
+
+/obj/item/weapon/shield/riot/makeshift/hit_reaction()
+	if(prob(30))
+		if(ishuman(loc))

```

Comment:
I'm not exactly positive on how to to do this part. I want it to only happen if hit with brute damage but yeah.